nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/documentation/ui/connector-definition/connector-definition.component.html

```diff
@@ -0,0 +1,204 @@
+                                        title="Collapse All"
+                                        (click)="collapseAllSteps()">
+                                        <i class="fa fa-compress fa-rotate-45"></i>
+                                    </button>
+                                </div>
+                            </div>
+                            <mat-accordion multi #stepsAccordion="matAccordion">
+                                @for (step of connectorDefinition.configurationSteps; track step.name) {
+                                    <mat-expansion-panel [expanded]="true">
+                                        <mat-expansion-panel-header>
+                                            <mat-panel-title>{{ step.name }}</mat-panel-title>
+                                        </mat-expansion-panel-header>
+                                        <div class="flex flex-col gap-y-4">
+                                            @if (step.description) {
+                                                <div class="tertiary-color">{{ step.description }}</div>
+                                            }
+                                            @if (step.documented) {
+                                                <mat-expansion-panel
+                                                    (opened)="loadStepDocumentation(connectorDefinition, step.name)">
+                                                    <mat-expansion-panel-header>
+                                                        <mat-panel-title>Step Documentation</mat-panel-title>
+                                                    </mat-expansion-panel-header>
+                                                    <div class="step-documentation">
+                                                        @if (isStepDocumentationLoading(step.name)) {
+                                                            <ngx-skeleton-loader count="3"></ngx-skeleton-loader>
+                                                        } @else if (getStepDocumentationError(step.name); as error) {
+                                                            <div class="error-color">{{ error }}</div>
+                                                        } @else if (getStepDocumentation(step.name); as documentation) {
+                                                            <markdown [data]="documentation"></markdown>
+                                                        }
+                                                    </div>
+                                                </mat-expansion-panel>
+                                            }
+                                            @if (hasPropertyGroups(step.propertyGroups)) {
+                                                @for (group of step.propertyGroups; track group.name) {
+                                                    <div class="property-group flex flex-col gap-y-2">
```

Comment:
Manage the bottom margin here with tailwind rather than in the scss file in the `.property-group` class

```suggestion
                                                    <div class="property-group flex flex-col gap-y-2 mb-4">
```